test/regress.c

```diff
@@ -1270,6 +1273,13 @@ test_signal_switchbase(void)
 	test_ok = 0;
 	base1 = event_init();
 	base2 = event_init();
+
+	if (!strcmp(event_base_get_signal_method(base1), "signalfd_signal") ||
+	    !strcmp(event_base_get_signal_method(base2), "signalfd_signal")) {
+		test_ok = 1;
```

Comment:
This doesn't work, not sure why. Using plain `tt_skip()` here gives an error:
```
../../source/test/test.sh -b EPOLL
Running tests:
EPOLL 
 test-eof: OKAY
 test-closed: OKAY
 test-weof: OKAY
 test-time: OKAY
 test-changelist: OKAY
 test-fdleak: OKAY
 test-dumpevents: OKAY
 regress: signal/signal_switchbase: 
  FAIL ../../source/test/regress_main.c:409: Legacy unit test failed
```

Reply:
Because the test set `test_ok` to `0` before - https://github.com/libevent/libevent/blob/9e346936d47a3a461f8ebaedabb2975bef3babf5/test/regress.c#L1270

And failed here - https://github.com/libevent/libevent/blob/9e346936d47a3a461f8ebaedabb2975bef3babf5/test/regress_main.c#L408